InterviewFlashCards/QuestionsViewController.swift

```diff
@@ -8,52 +8,81 @@
 
 import UIKit
 
-enum SectionQuestionType: Int {
+enum SectionQuestionType {
     case iOSTechnical
     case DataStructures
     case Algorithms
-    
+
 }
 
 class QuestionsViewController: UIViewController {
-    
+
     // MARK: IBOutlets
     @IBOutlet weak var prevButton: UIButton!
     @IBOutlet weak var nextButton: UIButton!
     @IBOutlet weak var questionImageView: UIImageView!
     @IBOutlet weak var questionLabel: UILabel!
     @IBOutlet weak var answerButton: UIButton!
-    
+
     // MARK: Private Properties
-    private let queryManager = QueryManager()
-    private var flashCards = [IFCFlashCard]()
+    private(set) var flashCards = [IFCFlashCard]()
     private var currentIndex = 0
     private var section: SectionQuestionType = .DataStructures
-    
+
     // MARK: Public Properties
     var sectionName = ""
-    
+    var dataGenerator: Networking?
```

Comment:
so a `mock` is an object that acts in the place of another, usually used to avoid having dependency issues defined by the original object. In this case the object being mocked out is the `QueryManager`. The `QueryManager` makes network calls, which you don't want in tests. So we create a `mock` and pass that in, giving us full control of what data gets sent to the view controller.